### PR TITLE
update google addon analytics package

### PR DESCRIPTION
**Pull request by jeffvg**
Fixes # .

#### Changes proposed in this Pull Request:
- storybook addon google analytics upgraded to 6.2.9
- dev branch to report google analytics
- 

#### To Test:
- 
go to 
https://brightlayer-ui-components.github.io/angular-dev/?path=/info/components-info-list-item--readme

the open google analytic
https://analytics.google.com/analytics/web/?authuser=0#/realtime/rt-overview/a139646200w231855793p217853416/

navigate around

### CODE REVIEW

File: demos/storybook/package.json [hunk 1]
```diff
@@ -27,12 +27,12 @@
         "@angular/platform-browser-dynamic": "~12.2.4",
         "@brightlayer-ui/angular-themes": "^6.3.0-beta.0",
         "@brightlayer-ui/colors": "^3.0.1",
+        "@brightlayer-ui/icons": "^1.7.1-beta.1",
```
Comment (emclaug2) [inline demos/storybook/package.json:30]: Why are these beta packages?  Since you're in here can you update these to be 1.8.1 ?

**Review by emclaug2 — APPROVED** [review]
Approving this, just see the one comment above.

File: demos/storybook/package.json (hunk 1 above)
Comment (jeffvg) [reply demos/storybook/package.json:30]: done. Also upgraded icons-svg

**Comment by emclaug2** [comment]
@jeffvg You'll need to delete the yarn.lock in demos/storybook and re-add it.

**Pull request merged**